### DISCUSSION TITLE
Valhalla va-t-il corriger l'erreur à 1 milliards de dollars ? - Marseille Java User Group

### DISCUSSION THREAD

**Discussion by giscus** [marsjug/marsjug.github.com#43] [General]
# Valhalla va-t-il corriger l'erreur à 1 milliards de dollars ? - Marseille Java User Group

Le Mars JUG réunit régulièrement la communauté Java de Marseille et de sa région autour d'intervenants qui viennent nous parler de la plateforme Java et de son écosystème.

http://localhost:8080/reunions/valhalla-va-t-il-corriger-l-erreur-1-milliards-de-dollars

**Comment by sherine-k** [comment]
ok
